torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1675,24 +1676,37 @@ def solve(self):
                 if s not in self._substitutions or not sympy.checksol(congruence, {s: self._substitutions[s]}):
                     self._dynamic_results.add(self._dcp.doprint(sympy.Eq(congruence, 0)))
 
-    def prettify_results(self):
+    def prettify_results(self, original_signature: inspect.Signature):
+        # Note: Model inputs are wrapped as LocalSource in dynamo.
+        # LocalSource.name() wraps the name with L[""]. We use regular
+        # expression to do the replacement to avoid traversing up
+        # the source hierarchy manually.
+        def unwrap_local_source(source_name):
+            return re.sub(r"L\['(.+?)'\]", r'\1', source_name)
+
         buf = ""
+        indent = 4 * " "
         if self._static_results:
+            sorted_static_results = sorted(self._static_results)
```

Comment:
Is it sorted by tensor first then by dim?

Reply:
It's just sorted lexicographically to have deterministic results.